src/components/Dashboard/FirmsAddressList.vue

```diff
@@ -133,7 +133,10 @@ export default class FirmsAddressList extends Mixins(CommonMixin, CountriesProvi
   @Getter getBusinessAddress!: OfficeAddressIF
 
   // Business
-  get businessAddress (): OfficeAddressIF {
+  get businessAddress (): OfficeAddressIF|null {
+    if (this.getBusinessAddress && this.getBusinessAddress.deliveryAddress &&
```

Comment:
You can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here to simplify things a bit.

Also on line 139.